### PR TITLE
AB#3930 -- Updated translation for terms & conditions en/fr text

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Updated the translations on the terms & conditions page for both en/fr text (from figma).
Added in italics

### Related Azure Boards Work Items
[AB#3930](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3930)

### Screenshots (if applicable)
image

### CODE REVIEW

File: frontend/public/locales/en/apply.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
     "terms-and-conditions-of-use": {
       "summary": "Terms and Conditions of Use",
-      "online-application-legal-terms": "PLEASE READ CAREFULLY. These Terms for the Canadian Dental Care Plan (CDCP) Online Application contain information about your legal rights and obligations and regulate your use of the Online Application. These Terms should be read in conjunction with the <canadaTermsConditions>Canada.ca website Terms and Conditions</canadaTermsConditions>. In case they are differing, the present Terms take precedence over Canada.ca Terms and Conditions.",
+      "online-application-legal-terms": "<strong>PLEASE READ CAREFULLY</strong>. These Terms for the Canadian Dental Care Plan (CDCP) Online Application contain information about your legal rights and obligations and regulate your use of the Online Application. These Terms should be read in conjunction with the <canadaTermsConditions>Canada.ca website Terms and Conditions</canadaTermsConditions>. In case they are differing, the present Terms take precedence over Canada.ca Terms and Conditions.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/apply.json:26]: Period of `PLEASE READ CAREFULLY` should be inside `strong` tags.

File: frontend/public/locales/fr/apply.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
...
+      "online-application-legal-terms": "<strong>VEUILLEZ LIRE ATTENTIVEMENT</strong>. Les présentes conditions d'utilisation de la demande en ligne du Régime canadien de soins dentaires (RCSD) contiennent des renseignements sur vos droits et obligations juridiques et régissent votre utilisation de la demande en ligne. Les présentes conditions d'utilisation doivent être lues en parallèle avec celles du site Web Canada.ca. En cas de divergence, les présentes conditions d'utilisation ont préséance sur celles du site Web <canadaTermsConditions>Canada.ca</canadaTermsConditions>.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/apply.json:26]: Period of `VEUILLEZ LIRE ATTENTIVEMENT` should be inside `strong` tags.

Comment (sebastien-comeau) [reply frontend/public/locales/fr/apply.json:26]: `En cas de divergence, les présentes conditions générales d'utilisation ont préséance sur celles du site Web Canada.ca.` is still present in Figma unless I'm not looking at the right mockup.

File: frontend/public/locales/en/apply.json [hunk 3]
```diff
@@ -48,7 +48,8 @@
...
-        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the Department of Health Act, the Dental Care Measures Act, the Department of Employment and Social Development Act, and is protected by the provisions of the Privacy Act. ESDC carries out this application process using services from a third-party provider, Microsoft Corporation, which may be subject to foreign laws. For more information, see <microsoftDataPrivacyPolicy>Microsoft Data Privacy Policy & Privacy Principles</microsoftDataPrivacyPolicy>."
+        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and is protected by the provisions of the <i>Privacy Act</i>.",
```
Comment (nicholas-ly) [inline frontend/public/locales/en/apply.json:51]: Referencing the title of a legal document should be done using the [citation element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite)

```suggestion
        "collection-use": "The collection, use, and disclosure of your personal information is authorized by, among other laws, the <cite>Department of Health Act</cite>, the <cite>Dental Care Measures Act</cite>, the <cite>Department of Employment and Social Development Act</cite>, and is protected by the provisions of the <cite>Privacy Act</cite>.",
```

(Note that this same comment can be applied wherever `<i>` is used).

File: frontend/public/locales/fr/apply.json [hunk 4]
```diff
@@ -47,38 +47,39 @@
...
       }
     },
     "apply": {
-      "application-participation": "(FR) Participation in the Canadian Dental Care Plan is voluntary. However, if you do not provide your personal information in your application, you cannot be considered for the Canadian Dental Care Plan.",
-      "application-consent": "En poursuivant la demande, vous acceptez aux conditions générales d'utilisation ci-dessus et à l'énoncé de confidentialité. ",
+      "application-participation": "La participation au Régime canadien de soins dentaires est volontaire. Toutefois, si vous ne fournissez pas vos renseignements personnels dans votre demande, votre admissibilité au Régime canadien de soins dentaires ne pourra pas être validée.",
+      "application-consent": "En poursuivant cette demande, vous reconnaissez avoir lu et accepté les modalités et l'énoncé de confidentialité cidessus. De plus, vous consentez à ce que vos renseignements personnels soient communiqués aux fins susmentionnées.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/apply.json:82]: `cidessus` should be `ci-dessus`

Comment (sebastien-comeau) [reply frontend/public/locales/fr/apply.json:82]: I added a [comment](https://www.figma.com/design/pXiZz8YkzcMY5S5aq0B4ce?node-id=12172-58867#840741893) on Figma.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en/apply.json (hunk 3 above)
Comment (sebastien-comeau) [reply frontend/public/locales/en/apply.json:51]: I don't disagree but [Canada.ca - Terms and conditions](https://www.canada.ca/en/transparency/terms.html) doesn't use `cite` html tag.

Comment (nicholas-ly) [reply frontend/public/locales/en/apply.json:51]: Interesting. Seems like that Canada.ca page doesn't follow the  [Canada.ca Content Style Guide](https://design.canada.ca/style-guide/), which uses `cite` throughout.

Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply.json:51]: Updated to use `<cite>` tags

File: frontend/public/locales/en/apply.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply.json:26]: Moved period inside tags

File: frontend/public/locales/fr/apply.json (hunk 2 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Bad notepad++ comparison, re-added missing line. Moved period inside tags

**Review by sebastien-comeau — APPROVED** [review]

File: frontend/public/locales/fr/apply.json (hunk 4 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply.json]: Added hyphen

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**